### PR TITLE
feat: add skipLuhnValidation to card number element

### PR DESCRIPTION
**Pull request by lcschy**
## Description

- Add `skipLuhnValidation` prop for card numbers

## Testing required outside of automated testing?

- [x] Not Applicable

### Screenshots (if appropriate):

- [x] Not Applicable

## Rollback / Rollforward Procedure

- [x] Roll Forward
- [ ] Roll Back

## Reviewer Checklist

- [ ] Description of Change
- [ ] Description of outside testing if applicable.
- [ ] Description of Roll Forward / Backward Procedure
- [ ] Documentation updated for Change
-

### CODE REVIEW

File: src/utils/validation.ts [hunk 1]
```diff
@@ -62,7 +77,9 @@ const _cardNumberValidator = (value: string) => {
 const _expirationDateValidator = expirationDate;
 
 // eslint-disable-next-line @typescript-eslint/default-param-last
-const _maskValidator = (mask: Mask = [], value: string) => {
+const _maskValidator = (options: ValidatorOptions = {}, value: string) => {
+  const mask = options?.mask ?? [];
```
Comment (kevinperaza) [inline src/utils/validation.ts:81]: Did the default arg fail?

File: tests/components/CardVerificationCodeElement.test.tsx [hunk 2]
```diff
@@ -23,7 +23,7 @@ describe('CardVerificationCodeElement', () => {
   describe('mask', () => {
     test.each([3, 4])(
       'applies mask correctly with cvcLength = %d',
-      (cvcLength) => {
+      (cvcLength: any) => {
```
Comment (kevinperaza) [inline tests/components/CardVerificationCodeElement.test.tsx:26]: Nit: using `any` can undermine the benefits of using TS, even in tests. However, it's not a blocker.

File: src/utils/validation.ts (hunk 1 above)
Comment (lcschy) [reply src/utils/validation.ts:81]: You mean leaving `mask: Mask = []`? yeah that causes issues w/ the events

File: tests/components/CardVerificationCodeElement.test.tsx (hunk 2 above)
Comment (lcschy) [reply tests/components/CardVerificationCodeElement.test.tsx]: tbh idk how this even happened lol, fixed

**Review by kevinperaza — APPROVED** [review]

**Pull request merged**